equinox/nn/_embedding.py

```diff
@@ -89,3 +93,253 @@ def __call__(
                 "`eqx.nn.Embedding()(x)` should be called with a scalar index `x`. "
                 "Use `jax.vmap` if you would like to index with multiple values."
             )
+
+
+class RotaryPositionalEmbedding(Module):
+    embedding_size: int = field(static=True)
+    max_seq_len: Optional[int] = field(static=True)
+    freqs_cis: Optional[Complex[Array, "embedding_size/2"]] = field(static=True)
+
+    def __init__(
+        self,
+        embedding_size: int,
+        max_seq_len: Optional[int] = None,
+        *,
+        key: Optional[PRNGKeyArray] = None,
+        **kwargs,
+    ):
+        """**Arguments:**
+
+        `RotaryPositionalEmbedding` requires:
```

Comment:
By "docs on the attention page", are you referring to the file docs/api/nn/attention.md? 

If so, sure, I'll add it. But might it not be a bit confusing, since the classes are implemented under `_embedding.py` yet their docs are under `attention.md`?

Reply:
So the existing embedding docs are for a different kind of embedding, namely lookup tables. We overload the word "embedding" a lot in ML!
So I'm thinking of keeping these together with attention, since that's where they used.